packages/web-components/src/progress-bar/progress-bar.styles.ts

```diff
@@ -98,7 +99,7 @@ export const styles = css`
       background-color: CanvasText;
     }
     .indicator,
-    :host(:is([validation-state='success'], [validation-state='warning'], [validation-state='error'])) .indicator {
+    :host(:is(${successState}, ${warningState}, ${errorState})) .indicator {
```

Comment:
Unnecessary `:is()` wrapper?

```suggestion
    :host(${successState}, ${warningState}, ${errorState}) .indicator {
```

Reply:
Interestingly enough, it's necessary here to get the selector chain correct.